modules/administration-guide/pages/configuring-a-user-namespace.adoc

```diff
@@ -0,0 +1,96 @@
+:_content-type: PROCEDURE
+:description: Configuring a user namespace
+:keywords: administration guide, configuring, user, namespace
+:navtitle: Configuring a user namespace
+:page-aliases:
+
+[id="configuring-a-user-namespace"]
+= Configuring a user namespace
+
+This procedure walks through the process of using the {prod-short}
+to replicate `ConfigMaps`, `Secrets` and `PersistentVolumeClaim` from `{prod-namespace}` namespace to numerous
+user-specific namespaces. The operator automates the synchronization of important configuration
```

Comment:
Which operator is it? Does it have a name e.g. Dev Spaces Operator?

Reply:
It is `{prod}` operator